test/test_other.py

```diff
@@ -15169,16 +15169,21 @@ def test_empath_split(self):
       #include <iostream>
       void foo() { std::cout << "foo" << std::endl; }
     ''')
-    create_file('path_list', r'''
+    create_file('path_list.txt', r'''
+      myapp
```

Comment:
nit: Can you end these lines with `:` maybe?   I feel like it would make them easier to read if I could clearly see the different between module names and filenames.

Reply:
I'd like to make the file format similar to that of wasm-split's multi split manifest, which does not have colons: https://github.com/WebAssembly/binaryen/blob/main/test/lit/wasm-split/multi-split.wast.manifest

Reply:
> I don't mind much, but wasm-split's manifest file already has some users. If we decide to do it, I think we can give some internal clients a heads-up though. WDYT? @tlively

Making improvements to the manifest file format sounds good to me. I imagine we might want to make more changes in the future as well, for example to include information about dependencies between modules.